sdk/communication/Azure.Communication.Chat/src/ChatThreadClient.cs

```diff
@@ -375,67 +416,101 @@ public virtual Response AddMembers(IEnumerable<ChatThreadMember> members, Cancel
             }
         }
 
-        /// <summary> Gets the members of a thread asynchronously. </summary>
+        /// <summary> Gets the participants of a thread asynchronously. </summary>
+        /// <param name="skip"> Skips participants up to a specified position in response. </param>
         /// <param name="cancellationToken"> The cancellation token to use. </param>
         /// <exception cref="RequestFailedException">The server returned an error. See <see cref="Exception.Message"/> for details returned from the server.</exception>
-        public virtual AsyncPageable<ChatThreadMember> GetMembersAsync(CancellationToken cancellationToken = default)
+        public virtual AsyncPageable<ChatParticipant> GetParticipantsAsync(int? skip = null, CancellationToken cancellationToken = default)
```

Comment:
Do we want to expose the skip parameter? Unless we have specific scenario, I wonder if it makes sense to not expose this for now. Thoughts?

Reply:
I would prefer to have it consistent with the other pagination operations. I could also see some potential UI on top of this on contoso where they go throit once (or maybe just want the count), and get certain pages on demand after.